code/modules/surgery/organs/blood.dm

```diff
@@ -126,255 +80,199 @@
 					blood_max += W.damage / 4
 			if(temp.open)
 				blood_max += 2  //Yer stomach is cut open
```

Comment:
You may have to change the above to what TG has, lest this be insanely OP or do nothing at all.